light-client/Cargo.toml

```diff
@@ -45,7 +44,8 @@ thiserror = "1.0.15"
 tokio = "0.2.20"
 
 [dev-dependencies]
+tendermint-testgen = { path = "../testgen"}
```

Comment:
This seemed to not work for me when using testgen data structure inside src/test.rs

Reply:
Yes, but I moved the `From` instance to convert between `TMLightBlock` and `AnonBlock` to a simple function `tests/model_based.rs` to avoid introducing the dependency.

It's unfortunate that one cannot define a test-only module within a crate and then reuse it in standalone tests (ie. those in the `tests` folder).

Until then, that's the only way I saw to avoid adding `testgen` as a regular dependency.

Reply:
I think @romac is very right to point out that testgen should be a dev-dependency. What this shows, is that a bit of refactoring is needed.